apps/main/src/GithubApp/GithubApp.tsx

```diff
@@ -13,10 +13,14 @@ import { getDateCondition, getLanguageCondition, getLicenseCondition, getReposit
 export const GithubApp = () => {
   const { handleUpdateToken, isTokenExist } = useInputTokenForGraphql()
 
-  const [license, setLicense] = useState<string>('')
-
-  const [repositoryNameValue, setRepositoryName] = useState<string>('')
-  const [repositoryName] = useDebouncedValue(repositoryNameValue, 300)
+  const form = useForm({
+    initialValues: {
+      license: '',
+      repositoryName: '',
+    },
+  })
+  const license = form.values.license
+  const repositoryName = form.values.repositoryName
```

Comment:
I see one more thing:
- in previous version the `repositoryName` fields was debounced
- it need to reduce requests to the GitHub GraphQL API
and the behaviour of app are different now

To compare behaviour of app use its link please: https://dipiash.github.io/nx-vite-react-ts-mantine-boilerplate/

Might be in this case the good thing to use debounce function like this:
```typescript
const debounce = (fn: Function, ms = 300) => {
  let timeoutId: ReturnType<typeof setTimeout>

  return function (this: any, ...args: any[]) {
    clearTimeout(timeoutId)
    timeoutId = setTimeout(() => fn.apply(this, args), ms)
  }
}
```
for onChange in Input field: https://github.com/dipiash/nx-vite-react-ts-mantine-boilerplate/pull/57/files#diff-b517f49c6141b06f0d84e515e08d53dab8aa7df7e40f3ea61cfe9fd1834a8a7eR21

Reply:
Hey, Can we not use the useDebouncedValue hook from mantine/hooks as used in the previous version?
```tsx
  import { useDebouncedValue } from '@mantine/hooks'
  ...
  const form = useForm({
    initialValues: {
      license: '',
      repositoryName: '',
    },
  })

  const license = form.values.license
  const repositoryNameValue = form.values.repositoryName

  const [repositoryName] = useDebouncedValue(repositoryNameValue, 300)
```

Reply:
If it works like in previous variant you can do it